cogs/moderation.py

```diff
@@ -1,10 +1,33 @@
 import asyncio
+from collections import Counter
+from cogs.utils.embedHandler import failure, info, success
 
 import discord
-from cogs.utils.embedHandler import failure, info, success
 from discord.ext import commands
 
-
+# Converters
+
+def can_execute_action(ctx, user, target):
+    return user.id == ctx.bot.owner_id or \
+           user == ctx.guild.owner or \
+           user.top_role > target.top_role
+
+class MemberNotFound(Exception):
+    pass
+
+async def resolve_member(guild, member_id):
+    member = guild.get_member(member_id)
+    if member is None:
+        if guild.chunked:
+            raise MemberNotFound()
+        try:
+            member = await guild.fetch_member(member_id)
+        except discord.NotFound:
+            raise MemberNotFound() from None
```

Comment:
```suggestion
            raise MemberNotFound()
```
What is `from None` it doesn't seem necessary